docs/getting-started-guides/minikube.md

```diff
@@ -191,7 +191,7 @@ Minikube sets this context to default automatically, but if you need to switch b
 
 `kubectl config use-context minikube`,
```

Comment:
@netroby now that these sentences are split up into multiple sentences, this comma can be deleted.